src/Pelagos/Bundle/AppBundle/Resources/views/DataDiscovery/template-dataset-row.html.twig

```diff
@@ -3,11 +3,10 @@
     <td class="info">
         <span id="container-dataset-authors"><span id="dataset-authors"></span>. </span>
         <span id="container-dataset-year"><span id="dataset-year"></span>. </span>
-        <a href="{{ path('pelagos_app_ui_dataland_default', {'udi' : 'placeholder-udi'}) }}" target="_blank" id="dataset-title">
-        </a>.
+        <span id="dataset-title">. </span>
         <span id="container-dataset-research-group"><span id="dataset-research-group"></span>. </span>
-        <span id="container-dataset-doi"><span id="label-dataset-doi">DOI: </span><a href="http://dx.doi.org/{{ 'placeholder-doi' }}" target="_blank" id="dataset-doi"></a>. </span>
-        <span id="container-dataset-udi"><span id="label-dataset-udi">UDI: <a href="{{ path('pelagos_app_ui_dataland_default', {'udi' : 'placeholder-udi'}) }}" target="_blank" id="dataset-udi"></a>. </span></span>
+        <span id="container-dataset-doi"><span id="label-dataset-doi">DOI: </span><span id="dataset-doi">. </span>
```

Comment:
Maybe some of these spans need to be on new lines?

Reply:
yeah, cleaned this up